src/Sylius/Bundle/CoreBundle/Sitemap/Model/SitemapUrl.php

```diff
@@ -0,0 +1,150 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+ 
+namespace Sylius\Bundle\CoreBundle\Sitemap\Model;
+
+use Sylius\Bundle\CoreBundle\Sitemap\Renderer\TemplateAware;
+
+/**
+ * @author Arkadiusz Krakowiak <arkadiusz.krakowiak@lakion.com>
+ */
+class SitemapUrl implements SitemapUrlInterface, TemplateAware
+{
+    /**
+     * @var string
+     */
+    private $loc;
+
+    /**
+     * @var \DateTime
+     */
+    private $lastmod;
+
+    /**
+     * @var string
+     */
+    private $changefreq;
+
+    /**
+     * @var float
+     */
+    private $priority;
+
+    /**
+     * @var string
+     */
+    private $template;
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getLoc()
+    {
+        return $this->loc;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function setLoc($loc)
+    {
+        $this->loc = $loc;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getLastmod()
+    {
+        return $this->lastmod;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function setLastmod(\DateTime $lastmod)
+    {
+        $this->lastmod = $lastmod;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getChangefreq()
+    {
+        return $this->changefreq;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function setChangefreq($changefreq)
```

Comment:
I'd say we should use normal names in the object, because someone who does not know sitemap structure will have hard time figuring out what they mean. We can configure serializer to use appropriate names correctly. Agree?